jenkins/run-unittests

```diff
@@ -7,22 +7,33 @@ source $(dirname "${0}")/python-setup.sh
 
 typeset -i errors=0
 
-printf -- "black --check ...\n"
+printf -- "\n\n\nrun-unittests started ...\n"
+
+printf -- "\n\nblack --check ...\n"
 black --check .
 black_sts=${?}
 if [[ ${black_sts} -ne 0 ]]; then
     printf -- "FAILED: black --check .\n" >&2
     (( errors++ ))
 fi
 
-printf -- "\nflake8 --verbose ...\n"
+printf -- "\n\nflake8 --verbose ...\n"
 flake8 --verbose .
 flake_sts=${?}
 if [[ ${flake_sts} -ne 0 ]]; then
     printf -- "FAILED: flake8 --verbose .\n" >&2
     (( errors++ ))
 fi
-printf -- "\n"
+
+printf -- "\n\npytests ...\n"
+pytest server/lib/pbench/test/unit
+pytest_sts=${?}
+if [[ ${pytest_sts} -ne 0 ]]; then
+    printf -- "FAILED: pytest server/lib/pbench/test/unit\n" >&2
+    (( errors++ ))
+fi
```

Comment:
So...does this replace the removal from the `tox` command line?

Reply:
Yes, see the commit message, we can't use `py3-server` with `--current-env`.

Reply:
I'll update the PR description as well.